### PR TITLE
DOC: fix includes

### PR DESCRIPTION
**Pull request by afeld**
- Added missing `.` to the `.. include`s
- Put the includes in a folder to exclude from Sphinx, to avoid `:orphan:` being written to the HTML

Before:

Screen_Shot_2020-12-30_at_7_08_52_PM

After:

Screen Shot 2020-12-30 at 7 07 41 PM

---

- [ ] ~~closes #xxxx~~
- [ ] tests added / passed
- [ ] ~~passes `black pandas`~~
- [ ] ~~passes `git diff upstream/master -u -- "*.py" | flake8 --diff`~~
- [ ] ~~whatsnew entry~~

### CODE REVIEW

**Comment by jreback** [comment]
small linting error but the doc build passed: https://github.com/pandas-dev/pandas/pull/38839/checks?check_run_id=1628340684

File: doc/source/conf.py [hunk 1]
```diff
@@ -68,7 +68,11 @@
     "contributors",  # custom pandas extension
 ]
 
-exclude_patterns = ["**.ipynb_checkpoints"]
+exclude_patterns = [
+    "**.ipynb_checkpoints",
+    # https://github.com/sphinx-doc/sphinx/issues/1965#issuecomment-124732907
+    "**/includes/**"
```
Comment (jorisvandenbossche) [inline doc/source/conf.py:74]:
```suggestion
    "**/includes/**",
```

File: doc/source/conf.py [hunk 2]
```diff
@@ -68,7 +68,11 @@
     "contributors",  # custom pandas extension
 ]
 
-exclude_patterns = ["**.ipynb_checkpoints"]
+exclude_patterns = [
+    "**.ipynb_checkpoints",
+    # https://github.com/sphinx-doc/sphinx/issues/1965#issuecomment-124732907
```
Comment (jorisvandenbossche) [inline doc/source/conf.py:73]: Can you in addition to the link also put a short explainer what this is about?

**Review by jorisvandenbossche — COMMENTED** [review]
Thanks!

File: doc/source/conf.py (hunk 1 above)
Comment (afeld) [reply doc/source/conf.py]: Done.

File: doc/source/conf.py (hunk 2 above)
Comment (afeld) [reply doc/source/conf.py:74]: Done.

**Comment by jreback** [comment]
thanks @afeld (pls check dev docs for rendered results in a while)

**Comment by afeld** [comment]
They look good!

- https://pandas.pydata.org/pandas-docs/dev/getting_started/intro_tutorials/03_subset_data.html
- https://pandas.pydata.org/pandas-docs/dev/getting_started/intro_tutorials/05_add_columns.html

**Pull request merged**